libs/libjson/_internal/Source/internalJSONNode.h

```diff
@@ -156,8 +156,8 @@ class internalJSONNode {
 	   DECL_CAST_OP(short)
 	   DECL_CAST_OP(int)
 	   DECL_CAST_OP(long)
+	   DECL_CAST_OP(long long)
 	   #ifndef JSON_ISO_STRICT
-		  DECL_CAST_OP(long long)
```

Comment:
why do you need to take this out of the ISO_STRICT? are we compiling with the strict?